src/headers/debug_op.h

```diff
@@ -38,6 +38,19 @@
 #endif
 #endif
 
+/**
+ * @brief Enumeration of the different log levels.
+ *
+ */
+typedef enum {
+    DEBUG_LOG = 0,
+    DEBUG_VERBOSE_LOG,
+    INFO_LOG,
+    WARNING_LOG,
+    ERROR_LOG
+} log_level_t;
```

Comment:
The numeric constant could be ordered in the same order of importance.
I think DEBUG_VERBOSE_LOG is less important than DEBUG_LOG. So the enum order could be:
    DEBUG_VERBOSE_LOG = 0,
    DEBUG_LOG,
    INFO_LOG,
    WARNING_LOG,
    ERROR_LOG

Reply:
This has been changed in the last commit.